packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -213,6 +220,18 @@ export function RepositoryMixin<T extends MixinTarget<Application>>(
       }
     }
 
+    mountComponentModels(component: {models?: Class<Model>[]}) {
+      if (!component.models) return;
+      for (const m of component.models) {
+        this.model(m);
+      }
+    }
+
+    getComponentInstance<T extends object>(component: Class<unknown>) {
+      const componentKey = `${CoreBindings.COMPONENTS}.${component.name}`;
+      return this.getSync<T>(componentKey);
```

Comment:
Please revert this to be in favor of https://github.com/strongloop/loopback-next/pull/5477/commits/1cf38b14016dd60bb3ff43b3b2f1ede7dd4e5972#r426750582